### PR TITLE
Feat/add lottie player

### PR DESCRIPTION
**Pull request by s-haensch**
# 💬 Proposed changes

This PR adds a lottie animation to the landing page using `lottie-web` along with some custom components and hooks. Also adds new sections to the landing page.

# ✨ Preview

### Desktop
homepage-desktop_1000

### Mobile
homepage-mobile_700

# 🖥📱 Browser / Device tests

- Firefox Nightly (109.0a1), Windows 11
- Chrome 108.0.5359.100, Windows 11

### CODE REVIEW

**Comment by s-haensch** [comment]
@vogelino @dnsos I have applied the requested changes. Ready for re-review.

**Review by dnsos — COMMENTED** [review]
Thanks for the changes @s-haensch 

I'm making a few more adjustments and then merging this soon!

File: src/components/Intro/index.tsx [hunk 1]
```diff
@@ -11,15 +10,10 @@ export const Intro: FC = () => {
...
-          <div
-            className={classNames(
-              'relative',
-              'after:block after:absolute after:top-0 after:w-full after:h-full',
-              'after:bg-scale-good after:mix-blend-hue'
-            )}
-          >
-            <img src="/images/intro-image.jpg" alt="" />
-          </div>
+          <img
```
Comment (vogelino) [inline src/components/Intro/index.tsx:13]: We might want to user Next's optimized `Image` component

**Review by vogelino — APPROVED** [review]
Good job There too!

**Comment by dnsos** [comment]
Merging now because this is slightly blocking. There are a few finetunings left that will be done in the other ticket.

**Pull request merged**